echopype/consolidate/loc_utils.py

```diff
@@ -118,48 +118,32 @@ def check_loc_time_dim_duplicates(echodata: EchoData, time_dim_name: str) -> Non
         )
 
 
-def sel_interp(
-    ds: xr.Dataset,
+def sel_nmea(
     echodata: EchoData,
     loc_name: str,
-    time_dim_name: str,
     nmea_sentence: Union[str, None] = None,
     datagram_type: Union[str, None] = None,
 ) -> xr.DataArray:
     """
-    Selection and interpolation for a location variable.
+    Select location subset for a location variable based on NMEA sentence.
 
-    The selection logic is as follows, with 4 possible scenarios:
+    The selection logic is as follows, with 3 possible scenarios:
```

Comment:
still 4 in the list below? but maybe you meant the actual code that only has 3 cases? so the 2 do-nothing cases are just the `else`?

Reply:
Oh yes yes, still 4 but the 2 do-nothing cases are part of else. I will revert back to 4 since that makes more sense since I'm describing the scenarios and not exactly the code itself. Nice catch!